src/components/Details.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import Navbar from './Navbar';
+// import Europe from './Europ';
```

Comment:
- [ ] [OPTIONAL] Please consider removing commented pieces of code because it makes your codebase look cluttered. Please do the same all through your codebase 👍